draft-ietf-webtrans-http2.md

```diff
@@ -251,368 +278,411 @@ unidirectional, indicated by the second least significant bit of the
+     being ignored.
 
    Application Protocol Error Code:
-   : A variable-length integer containing the
-   application-specified reason the sender is ignoring the stream.
+   : A variable-length integer containing the application-specified reason the
+     sender is ignoring the stream.
 
 
-## WT_STREAM Frames {#WT_STREAM}
+## WT_STREAM Capsule {#WT_STREAM}
 
 *[WT_STREAM]: #
 
-WT_STREAM frames implicitly create a stream and carry stream data.
+WT_STREAM capsules implicitly create a WebTransport stream and carry stream
+data.
 
-The Type field in the WT_STREAM frame is either 0x0a or 0x0b.  This uses the
-same frame types as a QUIC STREAM frame with the OFF bit clear and the LEN bit
-set.  The FIN bit (0x01) in the frame type indicates that the frame marks the
-end of the stream in one direction.  Stream data consists of any number of 0x0a
-frames followed by a terminal 0x0b frame.
+The Type field in the WT_STREAM capsule is either 0x0a or 0x0b.  This uses the
+same capsule types as a QUIC STREAM frame with the OFF bit clear and the LEN
+bit set.  The FIN bit (0x01) in the capsule type indicates that the capsule
+marks the end of the stream in one direction.  Stream data consists of any
+number of 0x0a capsules followed by a terminal 0x0b capsule.
 
 ~~~
-WT_STREAM Frame {
+WT_STREAM Capsule {
   Type (i) = 0x0a..0x0b,
   Length (i),
   Stream ID (i),
   Stream Data (..),
 }
 ~~~
-{: #fig-wt_stream title="WT_STREAM Frame Format"}
+{: #fig-wt_stream title="WT_STREAM Capsule Format"}
 
-WT_STREAM frames contain the following fields:
+WT_STREAM capsules contain the following fields:
 
 Stream ID:
 : The stream ID for the stream.
 
 Stream Data:
-: Zero or more bytes of data for the stream.  Empty WT_STREAM frames MUST NOT be
-used unless they open or close a stream; an endpoint MAY treat an empty
-WT_STREAM frame that neither starts nor ends a stream as a session error.
+: Zero or more bytes of data for the stream.  Empty WT_STREAM capsules MUST NOT
+  be used unless they open or close a stream; an endpoint MAY treat an empty
+  WT_STREAM capsule that neither starts nor ends a stream as a session error.
 
-## WT_MAX_DATA Frames {#WT_MAX_DATA}
+## WT_MAX_DATA Capsule {#WT_MAX_DATA}
 
 *[WT_MAX_DATA]: #
 
-A WebTransport frame called WT_MAX_DATA is introduced to inform the peer of the
-maximum amount of data that can be sent on the WebTransport session as a
-whole.
+An HTTP capsule {{HTTP-DATAGRAM}} called WT_MAX_DATA (type=0x10) is introduced
+to inform the peer of the maximum amount of data that can be sent on the
+WebTransport session as a whole.
 
 ~~~
-WT_MAX_DATA Frame {
+WT_MAX_DATA Capsule {
   Type (i) = 0x10,
   Length (i),
   Maximum Data (i),
 }
 ~~~
-{: #fig-wt_max_data title="WT_MAX_DATA Frame Format"}
+{: #fig-wt_max_data title="WT_MAX_DATA Capsule Format"}
 
-WT_MAX_DATA frames contain the following field:
+WT_MAX_DATA capsules contain the following field:
 
    Maximum Data:
```

Comment:
This indent is unnecessary, right?

Reply:
Yes, just visually separated in the markdown, shouldn't have any effect on the output (don't have strong feelings either way for this)